README mwmichelpach1.md

```diff
@@ -1,5 +1,12 @@
 # Cloudera Director Public Scripts
 
+## Contributing Code
+
+In order to contribute code you must sign the ICLA(https://github.com/cloudera/livy/wiki/Individual-Contributor-License-Agreement-(ICLA)(For independant Contributors on their own behalf)  or CCLA(https://github.com/cloudera/livy/wiki/Corporate-Contributor-License-Agreement-(CCLA)(If you are contributing for a company or orgonization).
```

Comment:
I suggest that you split up the part about CCLA and have that be its own sentence. That way you don't need to put the "If you are contributing for a company or organization" part in parentheses, and can have that be the start of the sentence instead. Also, there's a typo in "orgonization".